flaskr/forms.py

```diff
@@ -1,4 +1,6 @@
 from flask_wtf import FlaskForm
+from flask_wtf.file import FileField, FileAllowed
```

Comment:
for file fields in form, we should be using 
FileField --> type of field
FileAllowed --> a validator
from flask-wtf